### ISSUE TITLE
"Hide the 'welcome back" doesn't work at all for me.

### ISSUE THREAD

**Issue by santropedro** [soscripted/sox#509]
**Describe the bug**
I see the "Welcome back" type messages as SOMETIMES as if I didn't have anything to stop them, while having activated its relevant sox checkbox.
Other times, I see them flash and hide in 1 second. In fact I just tried this by googling a random question I have never visited before in a site I haven't visited. The website hid it appropriately.
'll used to stop them with Ublock origin, now I do it with sox which is perfectly installed and functioning, as I continue to enjoy its other features.   I disabled Ublock on SE to not interfere with SOX.

**To Reproduce**
Steps to reproduce the behaviour:
1. Click sox gear on any SE page, check the "Hide the 'welcome back" option.
2. Eventually, the message "welcome back" will appear, specially if you don't upvote as frequently. If it doesn't appear, you might not have triggered it, it can take days, you can check its requirements [I see "Welcome back! If you found this question useful..." message very frequently](https://meta.stackexchange.com/a/91564)

**SOX errors logged in console**
CONSOLE PASTE OF THE ERRORED ONE: THE ONE THAT DIDN'T HID THE WELCOME BACK MESSAGE. 
```
71Third-party cookie will be blocked. Learn more in the Issues tab.
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: logging sox stored values --- 
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: access token set
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: SOX-apiCache
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: {"standOutDupeCloseMigrated":{"questions":[]},"quickAuthorInfo":{"users":[{"last_access_date":1332463774,"creation_date":1332371599,"user_type":"registered","user_id":181808,"sox_request_time":1712282899167},{"last_access_date":1328467505,"creation_date":1328311360,"user_type":"registered","user_id":178181,"sox_request_time":1712282899167},{"last_access_date":1711014972,"creation_date":1325467065,"user_type":"registered","user_id":176060,"sox_request_time":1712282899167},{"last_access_date":1386957755,"creation_date":1308113526,"user_type":"registered","user_id":163767,"sox_request_time":1712282899167},{"last_access_date":1398137237,"creation_date":1306394430,"user_type":"registered","user_id":162663,"sox_request_time":1712282899167},{"last_access_date":1633548796,"creation_date":1306359939,"user_type":"registered","user_id":162644,"sox_request_time":1712282899167},{"last_access_date":1497172368,"creation_date":1299064742,"user_type":"registered","user_id":158128,"sox_request_time":1712282899167},{"last_access_date":1679686595,"creation_date":1294347696,"user_type":"registered","user_id":155403,"sox_request_time":1712282899167},{"last_access_date":1509659772,"creation_date":1246060800,"user_type":"registered","user_id":245,"sox_request_time":1712282899167},{"last_access_date":1711421457,"creation_date":1246060800,"user_type":"registered","user_id":1288,"sox_request_time":1712282899167},{"last_access_date":1693601352,"creation_date":1246060800,"user_type":"registered","user_id":2598,"sox_request_time":1712282899167},{"last_access_date":1686356829,"creation_date":1685825051,"user_type":"registered","user_id":1373824,"sox_request_time":1712282918189},{"last_access_date":1709233950,"creation_date":1652916299,"user_type":"registered","user_id":1196463,"sox_request_time":1712282918189},{"last_access_date":1710366233,"creation_date":1643090748,"user_type":"registered","user_id":1135948,"sox_request_time":1712282918189},{"last_access_date":1712282805,"creation_date":1620801546,"user_type":"registered","user_id":997587,"sox_request_time":1712282918189},{"last_access_date":1712221696,"creation_date":1577353113,"user_type":"registered","user_id":674944,"sox_request_time":1712282918189},{"last_access_date":1712249831,"creation_date":1452868569,"user_type":"registered","user_id":314146,"sox_request_time":1712282918189},{"last_access_date":1712260204,"creation_date":1445592941,"user_type":"registered","user_id":308065,"sox_request_time":1712282918189},{"last_access_date":1712268194,"creation_date":1399373877,"user_type":"registered","user_id":260073,"sox_request_time":1712282918189},{"last_access_date":1712167927,"creation_date":1391206410,"user_type":"registered","user_id":249658,"sox_request_time":1712282918189},{"last_access_date":1712265967,"creation_date":1381493706,"user_type":"registered","user_id":236563,"sox_request_time":1712282918189},{"last_access_date":1711206006,"creation_date":1365376472,"user_type":"registered","user_id":218388,"sox_request_time":1712282918189},{"last_access_date":1709651671,"creation_date":1343856493,"user_type":"registered","user_id":192171,"sox_request_time":1712282918189},{"last_access_date":1712196242,"creation_date":1329452200,"user_type":"registered","user_id":179041,"sox_request_time":1712282918189},{"last_access_date":1712254053,"creation_date":1319206730,"user_type":"registered","user_id":171199,"sox_request_time":1712282918189},{"last_access_date":1712263729,"creation_date":1288264906,"user_type":"registered","user_id":152859,"sox_request_time":1712282918189},{"last_access_date":1609802268,"creation_date":1603754876,"user_type":"registered","user_id":14525369,"sox_request_time":1712287888237},{"last_access_date":1712065423,"creation_date":1521794349,"user_type":"registered","user_id":9539287,"sox_request_time":1712287888237},{"last_access_date":1710360263,"creation_date":1402702844,"user_type":"registered","user_id":3739391,"sox_request_time":1712287888238},{"last_access_date":1712173514,"creation_date":1379272169,"user_type":"registered","user_id":2781889,"sox_request_time":1712287888238},{"last_access_date":1712250342,"creation_date":1326458311,"user_type":"registered","user_id":1147688,"sox_request_time":1712287888238},{"last_access_date":1675939776,"creation_date":1324135462,"user_type":"registered","user_id":1103595,"sox_request_time":1712287888238},{"last_access_date":1709650289,"creation_date":1348365188,"user_type":"registered","user_id":160871,"sox_request_time":1712289069646},{"last_access_date":1711381313,"creation_date":1301324293,"user_type":"registered","user_id":73829,"sox_request_time":1712289069646},{"last_access_date":1708414721,"creation_date":1470667532,"user_type":"registered","user_id":627521,"sox_request_time":1712321166325},{"last_access_date":1712255415,"creation_date":1461208432,"user_type":"registered","user_id":323179,"sox_request_time":1712322079302},{"last_access_date":1712312571,"creation_date":1400610804,"user_type":"registered","user_id":260841,
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: SOX-debug
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: true
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: SOX-lastVersionInstalled
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: 2.8.0
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: SOXSETTINGS
userscript.html?name=Stack-Overflow-Extras-SOX.user.js&id=62823c28-9338-4d93-8d18-1e7db2450c13:293 SOX: ["Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-localTimestamps","Appearance-metaChatBlogStackExchangeButton","Appearance-scrollToTop","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-hideWelcomeBackMessage","Appearance-openImagesAsModals","Comments-confirmNavigateAway","Comments-showCommentScores","Comments-onlyShowCommentActionsOnHover","Sidebar-hideCommunityBulletin","Sidebar-hideJustHotMetaPosts","Sidebar-hideHireMe","Sidebar-hideChatSidebar","Sidebar-hideLoveThisSite","Sidebar-linkedToFrom","Voting-betterCSS","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksPrivacy","Extras-shareLinksMarkdown","Extras-warnNotLoggedIn","Extras-copyCode","Extras-customMagicLinks"]
```

**Expected behaviour**
It should always hide the message, I see it too often.

**Screenshots/GIFs**
If I can capture later, I'll share one, the only goal would be to give credibility to this report.

**Environment**
Chrome Version 125.0.6422.112 (Official Build) (64-bit), SOX V2.8.0, I use it with Tampermonkey, in fact I installed it for this.

If this issue goes away after weeks, I'll delete the report, for now I would like to know if others have experienced it.

**Comment by shu8** [comment]
@santropedro Thanks for the bug report - please could you try installing the latest [dev version v2.8.13](https://github.com/soscripted/sox/raw/dev/sox.user.js) to see if this issue is fixed now?

It's probably going to take a while to test because of the nature of the banner, so I'll leave this open for another week or so to give you time!

**Comment by shu8** [comment]
Hi @santropedro - have you seen this message appear since the update?

**Comment by santropedro** [comment]
@shu8 I'm sorry for not replying to you. I kept getting the unwanted behaviour until today, so I became fed up some minutes ago. Then I saw your message. I had wrongly ignored it before, I apologize.

> could you try installing the latest [dev version v2.8.13](https://github.com/soscripted/sox/raw/dev/sox.user.js)

I just clicked on it and clicked update. I suppose its intalled now. Now, when I click on the gear icon of this sox "change this sox settings" the pop up that opens says "v2.8.14dev" so I assume that's updated enough as you needed me to do it. I'll see if it works.
